examples/src/java/org/pantsbuild/example/3rdparty_jvm.md

```diff
@@ -291,3 +291,237 @@ time pants is run (otherwise, pants will cache it):
             mutable=True),
       ],
     )
+
+Strict Dependencies
+-------------------
+
+### What is strict_deps?
+
+Strict_deps is a feature of JVM targets which controls the dependencies
+available on the compile classpath when that target is being compiled.
+
+When compiling a JVM target with strict_deps disabled, all transitive
+dependencies of that target are included on the compile classpath.
+
+When compiling a JVM target with strict_deps enabled, the only things on the
+compile classpath are those listed in its `dependencies` attribute and those
+listed in the `exports` attribute of any target's transitive `dependencies`.
```

Comment:
How about

> When compiling a JVM target with strict deps enabled, the compile classpath is more restricted. Instead of containing all transitive dependencies, only the direct dependencies and the targets exported by those dependencies are included in the compile classpath.